rundeckapp/grails-app/services/rundeck/services/JobSchedulesService.groovy

```diff
@@ -0,0 +1,261 @@
+package rundeck.services
+
+import com.dtolabs.rundeck.core.schedule.SchedulesManager
+import org.quartz.*
+import org.rundeck.app.components.schedule.TriggerBuilderHelper
+import rundeck.ScheduledExecution
+
+class JobSchedulesService implements SchedulesManager {
+
+    static transactional = false
+    SchedulesManager rundeckJobSchedulesManager
+
+    @Override
+    boolean isSchedulesEnable() {
+        return rundeckJobSchedulesManager.isSchedulesEnable()
+    }
+
+    @Override
+    Map handleScheduleDefinitions(String jobUUID, boolean isUpdate) {
+        return rundeckJobSchedulesManager.handleScheduleDefinitions(jobUUID, isUpdate)
+    }
+
+    @Override
+    TriggerBuilderHelper createTriggerBuilder(String jobName, String jobGroup, String cronExpression, int priority) {
+        return rundeckJobSchedulesManager.createTriggerBuilder(jobName, jobGroup, cronExpression, priority)
+    }
+
+    @Override
+    TriggerBuilderHelper createTriggerBuilder(String jobUUID, String cronExpression, String triggerName) {
+        return rundeckJobSchedulesManager.createTriggerBuilder(jobUUID, cronExpression, triggerName)
+    }
+
+    @Override
+    Date nextExecutionTime(String jobUUID, boolean require) {
+        return rundeckJobSchedulesManager.nextExecutionTime(jobUUID, require)
+    }
+
+    @Override
+    boolean isScheduled(String jobUUID) {
+        return rundeckJobSchedulesManager.isScheduled(jobUUID)
+    }
+
+    @Override
+    List getAllScheduled(String serverUUID = null, String project = null) {
+        return rundeckJobSchedulesManager.getAllScheduled(serverUUID, project)
+    }
+
+    @Override
+    boolean shouldScheduleExecution(String jobUUID) {
+        return rundeckJobSchedulesManager.shouldScheduleExecution(jobUUID)
+    }
+
+    @Override
+    void persistSchedulesToJob(String uuid, List schedules, Boolean shouldSchedule, String project) {
+        rundeckJobSchedulesManager.persistSchedulesToJob(uuid, schedules, shouldSchedule, project)
+    }
+
+    @Override
+    List getJobSchedules(String uuid, String project) {
```

Comment:
this method should also be removed from the SchedulesManager interface, it is not used in core